code/test/util.js

```diff
@@ -257,7 +427,234 @@ const printInnerTrees = ({ Debugger, innerTrees }) => {
   }
 }
 
+// makeWallet uses an index and unlocked web3.eth.account and creates and funds a ONEwallet
+const makeWallet = async (salt, deployer, effectiveTime, duration, setLastResortAddress) => {
+  if (duration === undefined) { duration = DURATION }
+  let lastResortAccount = web3.eth.accounts.create()
+  let lastResortAddress = lastResortAccount.address
+  if (setLastResortAddress === false) { lastResortAddress = ONEConstants.EmptyAddress }
+  const { wallet, seed, hseed, root, client: { layers } } = await createWallet({
+    salt: new BN(ONEUtil.keccak(salt)),
+    effectiveTime,
+    duration,
+    maxOperationsPerInterval: SLOT_SIZE,
+    lastResortAddress: lastResortAddress,
+    spendingLimit: ONE_ETH
+  })
+  // Fund wallet
+  await web3.eth.sendTransaction({
+    from: deployer,
+    to: wallet.address,
+    value: HALF_ETH
+  })
+  return { wallet, seed, hseed, root, layers, lastResortAddress: lastResortAccount.address }
+}
+
+// makeTokens makes test ERC20, ERC20Decimals9, ERC721, ERC1155
+const makeTokens = async (owner) => {
+  // create an ERC20
+  const testerc20 = await TestERC20.new(10000000, { from: owner })
+  // create an ERC721
+  const tids = [0, 1, 2, 3, 4, 5, 6, 7, 8, 9]
+  const uris = ['ipfs://test721/0', 'ipfs://test721/1', 'ipfs://test721/2', 'ipfs://test721/3', 'ipfs://test721/4', 'ipfs://test721/5', 'ipfs://test721/6', 'ipfs://test721/7', 'ipfs://test721/8', 'ipfs://test721/9']
+  const testerc721 = await TestERC721.new(tids, uris, { from: owner })
+  // create an ERC1155
+  const tids1155 = [0, 1, 2, 3, 4, 5, 6, 7, 8, 9]
+  const amounts1155 = [10, 20, 20, 20, 20, 20, 20, 20, 20, 100]
+  const uris1155 = ['ipfs://test1155/0', 'ipfs://test1155/1', 'ipfs://test1155/2', 'ipfs://test1155/3', 'ipfs://test1155/4', 'ipfs://test1155/5', 'ipfs://test1155/6', 'ipfs://test1155/7', 'ipfs://test1155/8', 'ipfs://test1155/9']
+  const testerc1155 = await TestERC1155.new(tids1155, amounts1155, uris1155, { from: owner })
+  return { testerc20, testerc721, testerc1155 }
+}
+
+// get OneWallet state
+const getONEWalletState = async (wallet) => {
+  Logger.debug(`getting State for: ${wallet.address}`)
+  const address = (wallet.address).toString()
+  const identificationKey = (await wallet.identificationKey()).toString()
+  // console.log(`identificationKey: ${JSON.stringify(identificationKey)}`)
+  const walletIdentificationKeys = await wallet.getIdentificationKeys()
+  // console.log(`walletIdentificationKeys: ${JSON.stringify(walletIdentificationKeys)}`)
+  let identificationKeys = []
+  try {
```

Comment:
might be easier to just remove the try block

Reply:
done 🙏